src/router.js

```diff
@@ -64,13 +61,7 @@ export default new Router({
       component: SaphyrView,
       meta: { pipeline: 'Saphyr' },
       children: [
-        { path: '', redirect: 'reception', meta: { page: 'Reception' } },
-        {
-          path: 'reception',
```

Comment:
We are removing saphyr reception and currently there isnt another way to import into saphyr. Looking at prod it appears saphyr hasnt been used since 2019. I think if/when they want to start using it again we should just implement it as an option in the general reception?

Reply:
Agreed. I will check with SciOps to see what they want to do about it. I also suspect because of code rot there will be things that aren't working.